superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -203,11 +213,40 @@ class Dashboard extends React.PureComponent {
     }
   }
 
+  componentWillUnmount() {
+    document.removeEventListener('keydown', this.handleKeydown);
+  }
+
   // return charts in array
   getAllCharts() {
     return Object.values(this.props.charts);
   }
 
+  handleKeydown(event) {
+    const controlOrCommand = event.ctrlKey || event.metaKey;
+    if (controlOrCommand) {
+      const isK = event.key === 'k' || event.keyCode === 83;
+      if (isK) {
+        this.setState({ showOmni: !this.state.showOmni });
+
+        // Get first input in the modal div
+        document
+          .getElementsByClassName('modal-dialog')[0]
+          .getElementsByTagName('input')[0]
+          .focus();
+
+        Logger.send(
```

Comment:
I'm not convinced of the value of logging all the keydowns. Each logging call will trigger an rpc to the server. 
https://github.com/apache/incubator-superset/blob/11a7ad00b7c44785752bf23ef90bcf4ebb58598e/superset/assets/src/logger.js#L59

Given how often people may type in a dashboard especially in the presence of filters, I don't think we should log these keydowns and potentially flood the server here.

Reply:
isn't this only sent for the toggling of the bar tho? since it's inside the `control + k` blocks?

Reply:
I want to verify that this feature is being used before continuing main reason i wanted to add the logger here